test/tests_for_db/ee2_MongoUtil_test.py

```diff
@@ -57,6 +58,39 @@ def test_init_ok(self):
         mongo_util = self.getMongoUtil()
         self.assertTrue(set(class_attri) <= set(mongo_util.__dict__.keys()))
 
+    def test_update_jobs_success(self):
+        job = get_example_job(status=Status.created.value)
+        job2 = get_example_job(status=Status.created.value)
+        job_id1 = job.save().id
+        job_id2 = job2.save().id
+        job_ids = [job_id1, job_id2]
+        scheduler_ids = [job.scheduler_id, job2.scheduler_id]
+        jobs_to_update = list(map(JobIdPair, job_ids, scheduler_ids))
+        self.getMongoUtil().update_jobs_to_queued(jobs_to_update)
+        job.reload()
+        job2.reload()
+        assert all(j.status == Status.queued.value for j in [job, job2])
+
+    def test_update_jobs_failure(self):
+        job = get_example_job(status=Status.created.value)
```

Comment:
In looking at this test again I think it should be structured a little differently, something like this:

```
for state in [<all states except created>]:
    jobid1 = create_job_with_state(created)
    jobid2 = create_job_with_state(state)
    assertRaises(mongoUtil.update(jobid1, jobid2), Exception("error text"))
```

E.g. individually check that the code will barf on all states except created.

Reply:
Alright, updated the test

Reply:
Actually consolidated the success and failure into one test

Reply:
:+1: